src/dali/plugin/pair_converter/ccxt.py

```diff
@@ -134,6 +136,11 @@
 # CSV Reader
 _GOOGLE_API_KEY: str = "google_api_key"
 
+# Djikstra weights
+# Priority should go to quote assets listed above, then other assets, and finally alternatives
+_STANDARD_WEIGHT: float = 50.0
+_ALTERNATIVE_MARKET_WEIGHT: float = 50.5
```

Comment:
Just a nit: I would avoid decimals, if possible, just for esthetics.

Reply:
Are the weights `float` in `prezzemolo` to facilitate calculations (as opposed to set values like these) then?

Reply:
Nothing wrong with using decimals: it was just an esthetic nit, not a strong opinion. Feel free to skip it if you prefer.